Gemfile

```diff
@@ -74,6 +81,9 @@ group :development do
   # Much nicer error pages https://github.com/BetterErrors/better_errors
   gem 'better_errors'
   gem 'binding_of_caller'
+  # Mina for deployment
+  #gem 'mina', require: false
+  #gem 'mina-puma', require: false,  github: 'untitledkingdom/mina-puma'
```

Comment:
why are these gems commented out? Can they be removed? Why are they in the 'development' group?